index.html

```diff
@@ -11,6 +11,10 @@
 <body>
   <div id="app">
     <h1>Vue.js Components Fundamentals</h1>
+    <click-counter></click-counter>
```

Comment:
should use v-for directive
for example: 

<click-counter v-for="count in 4" :key="count"></click-counter>